CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Add Scroll on filter sidebar items.
```

Comment:
I think that you should treat this change as an add, not a fix. What you think?

Reply:
But this is a bug, since were not showing the items.

Reply:
Its a bug fix because it should be done when the new filter was being made